bazel/foreign_cc/BUILD

```diff
@@ -156,6 +156,7 @@ envoy_cmake_external(
     name = "event",
     cache_entries = {
         "EVENT__DISABLE_OPENSSL": "on",
+        "EVENT__DISABLE_MBEDTLS": "on",
```

Comment:
Just to clarify, there is no CMake toggle required to enable wepoll over the previous default windows logic?

Reply:
Libevent backends are chosen only at runtime. A libevent consumer can specify which backends to avoid by either
1) setting EVENT_NO[METHOD_NAME] environment variables
2) calling the event_config_avoid_method API and using custom config